bpf/tests/ipsec_from_overlay_generic.h

```diff
@@ -1,19 +1,6 @@
-// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause) */
```

Comment:
here too.

Reply:
They were intentionally changed, otherwise CI checkpatch (`make -C bpf checkpatch`) will complain:

```
ERROR:C99_COMMENTS: do not use C99 // comments
#7: FILE: bpf/tests/ipsec_from_host_generic.h:1:
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
```